src/vs/workbench/browser/positronDataExplorer/components/dataExplorerPanel/components/dataExplorer.tsx

```diff
@@ -61,99 +61,99 @@ export const DataExplorer = () => {
-					return basicColumnHeaderWidth;
-				}
-
-				// Create a canvas and create a 2D rendering context for it to measure text.
-				const canvas = window.document.createElement('canvas');
-				const canvasRenderingContext2D = canvas.getContext('2d');
-
-				// If the 2D canvas rendering context couldn't be created, return the basic column
-				// header width.
-				if (!canvasRenderingContext2D) {
-					return basicColumnHeaderWidth;
-				}
-
-				// Set the column name width.
-				let columnNameWidth;
-				if (!columnName) {
-					columnNameWidth = 0;
-				} else {
-					// Measure the column name width using the font of the column name exemplar.
-					const columnNameExemplarStyle =
-						DOM.getComputedStyle(columnNameExemplar.current);
-					canvasRenderingContext2D.font = columnNameExemplarStyle.font;
-					columnNameWidth = canvasRenderingContext2D.measureText(columnName).width;
-				}
-
-				// Set the type name width.
-				let typeNameWidth;
-				if (!typeName) {
-					typeNameWidth = 0;
-				} else {
-					// Measure the type name width using the font of the type name exemplar.
-					const typeNameExemplarStyle = DOM.getComputedStyle(typeNameExemplar.current);
-					canvasRenderingContext2D.font = typeNameExemplarStyle.font;
-					typeNameWidth = canvasRenderingContext2D.measureText(typeName).width;
-				}
-
-				// Calculate return the column header width.
-				return Math.ceil(Math.max(columnNameWidth, typeNameWidth) + basicColumnHeaderWidth);
-			}
-		);
+			context.instance.tableDataDataGridInstance.horizontalCellPadding * 2;
 
 		// Calculate the width of a sort digit. The sort index is styled with font-variant-numeric
 		// tabular-nums, so we can calculate the width of the sort index by multiplying the width of
-		// a sort digit by the length of the sort index.
+		// a sort digit by 2.
 		const canvas = window.document.createElement('canvas');
 		const canvasRenderingContext2D = canvas.getContext('2d');
-		let sortIndexDigitWidth;
+		let sortIndexWidth;
 		if (!canvasRenderingContext2D) {
-			sortIndexDigitWidth = 0;
+			sortIndexWidth = 0;
 		} else {
 			const sortIndexExemplarStyle = DOM.getComputedStyle(sortIndexExemplar.current);
 			canvasRenderingContext2D.font = sortIndexExemplarStyle.font;
-			sortIndexDigitWidth = canvasRenderingContext2D.measureText('1').width;
+			sortIndexWidth = canvasRenderingContext2D.measureText('99').width;
 		}
 
-		// Set the sort index width calculator. Sort index widths are calculated.
-		context.instance.tableDataDataGridInstance.setSortIndexWidthCalculator(sortIndex =>
-			Math.ceil(sortIndex.toString().length * sortIndexDigitWidth)
-		);
+		/**
+		 * The column header width calculator.
+		 * @param columnName The column name.
+		 * @param typeName The type name.
+		 * @returns The column header width.
+		 */
+		const columnHeaderWidthCalculator = (columnName: string, typeName: string) => {
+			// Calculate the basic column header width. This allows for horizontal cell padding,
```

Comment:
We should have a note in the corresponding CSS file to update this calculation if the padding/width is changed there.

Reply:
I've done this.